pkg/features/podpresets.go

```diff
@@ -43,12 +42,12 @@ func activatePodPresets(feature *kubeoneapi.PodPresets, args *kubeadmargs.Args)
 	}
 	args.APIServer.ExtraArgs[pluginFlag] = newPlugins
 
-	currentConfig, hasConfig := args.APIServer.ExtraArgs[extraConfigFlag]
+	currentConfig, hasConfig := args.APIServer.ExtraArgs[runtimeConfigFlag]
 	var newConfig string
 	if hasConfig {
 		newConfig = currentConfig + "," + podPresetEnableValue
 	} else {
 		newConfig = podPresetEnableValue
 	}
-	args.APIServer.ExtraArgs[extraConfigFlag] = newConfig
+	args.APIServer.ExtraArgs[runtimeConfigFlag] = newConfig
```

Comment:
Don't we have a function in some other features that handle this appending automatically?

Reply:
Found it